docs/schema-loader.md

```diff
@@ -182,139 +219,184 @@ Create/Delete JDBC schemas
 
 ### Create namespaces and tables
 
-For using a config file (Sample config file can be found [here](https://github.com/scalar-labs/scalardb/blob/master/conf/database.properties)):
+To create a namespace and tables by using a properties file, run the following command, replacing `<VERSION>` with the version of ScalarDB that you're using and `<PATH_TO_PROPERTIES_FILE>` with the path to the properties file. If `--coordinator` is specified, a Coordinator table will be created.
+
 ```console
-$ java -jar scalardb-schema-loader-<version>.jar --config <PATH_TO_CONFIG_FILE> -f schema.json [--coordinator]
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config <PATH_TO_PROPERTIES_FILE> -f schema.json [--coordinator]
```

Comment:
> --config <PATH_TO_PROPERTIES_FILE> -f schema.json

`schema.json` is an arbitrary file, so it should be `<PATH_TO_SCHEMA_FILE>`or something?

BTW, for schema file, the example uses abbr. name `-f` while `--config` isn't abbreviated. A bit inconsistent?

Reply:
Oh, I didn't know we often used `--config` and `-f`. Your idea sounds good to me. Let's leave it as is in this PR.